### ISSUE TITLE
Error red colour does not meet minimum colour contrast

### ISSUE THREAD

**Issue by guermanc** [UKHomeOffice/design-system#351]
The GOV.UK Design System red (#D4351C) against the Home Office Design System background (#F1F1F1) does not meet minimum colour contrast requirements.

May need guidance for both internal and external services.

Suggested red is #AA2A16.

**Comment by tomyems** [comment]
Or we could change the page background to a different grey, if lots of the standard colours have issues.

**Comment by daniel-ac-martin** [comment]
Discussion opened for this. - https://github.com/UKHomeOffice/design-system/discussions/355
